e2e-tests/adapters/cypress/e2e/headers.cy.ts

```diff
@@ -114,7 +119,9 @@ describe("Headers", () => {
   })
 
   it("should contain correct headers for index page", () => {
-    cy.visit("/").waitForRouteChange()
+    cy.visit(
+      applyTrailingSlashOption(Cypress.config().baseUrl, TRAILING_SLASH)
```

Comment:
note, using `baseUrl` here to get full url, so that `applyTrailingSlashOption` can work correctly - otherwise it will always want to set `/` on "index" which would lead to redirect